config/routes.rb

```diff
@@ -3,11 +3,10 @@
     root to: 'dashboards#index', as: :authenticated_root
     get 'dashboard', to: 'dashboards#index', as: 'dashboard'
     post 'search', to: 'dashboards#search', defaults: { format: :js }
-    resources :pregnancies, only: [ :edit, :update ] do
+    resources :pregnancies, only: [ :create, :edit, :update ] do
```

Comment:
I see what you're saying, and I started to go through  with this change, but after switching the path names, e.g. edit_pregnancy_path to edit_patient_pregnancy_path, I realized that we'll also have to pass in an additional variable with each call to the path, i.e. edit_pregnancy_path(@pregnancy) becomes edit_patient_pregnancy_path(@patient, @pregnancy). 

Passing around those extra instance variables makes things a little messier, and we might be looking at things like edit_patient_pregnancy_path(@pregnancy.patient, @pregnancy) in a spot like pregnancies#update. Do you think it's still the smarter approach? I really can't decide which is better.